### ISSUE TITLE
3060M with 1 HDMI and 3 display port

### ISSUE THREAD

**Issue by dynamo-foundation** [arutar/FrankenDriver#4]
Any thoughts on getting Display Port operational?  Drivers are awesome by the way, thanks!

**Comment by arutar** [comment]
My video card has 2 hdmi ports and 2 DisplayPort ports. Both hdmi are working fine. DisplayPorts have not tested, I will check tomorrow

**Comment by arutar** [comment]
I found something interesting. Update from nvidia to fix the problem. Very scary...
https://www.nvidia.com/content/DriverDownloads/confirmation.php?url=/Windows/uefi/firmware/1.0/NVIDIA_DisplayID_Firmware_Updater_1.0-x64.exe&firmware=1&lang=us&type=Other&ncid=afm-chs-44270&ranMID=44270&ranEAID=a1LgFw09t88&ranSiteID=a1LgFw09t88-kdb0gwuDoGCsXECME0INPw

**Comment by arutar** [comment]
I would be afraid to run this...

**Comment by dynamo-foundation** [comment]
already tried that one the firmware cannot be updated.  also nvflash doesnt work although we can read it with a 341a set to 1.8v

Sent from my iPhone

On Dec 20, 2022, at 6:26 PM, arutar ***@***.***> wrote:

﻿

I found something interesting. Update from nvidia to fix the problem. Very scary...
https://www.nvidia.com/content/DriverDownloads/confirmation.php?url=/Windows/uefi/firmware/1.0/NVIDIA_DisplayID_Firmware_Updater_1.0-x64.exe&firmware=1&lang=us&type=Other&ncid=afm-chs-44270&ranMID=44270&ranEAID=a1LgFw09t88&ranSiteID=a1LgFw09t88-kdb0gwuDoGCsXECME0INPw

**Comment by dynamo-foundation** [comment]
Were you able to get the DP outputs to work?

**Comment by arutar** [comment]
it looks like I have all ports working with reboot and shutdown just checked

**Comment by dynamo-foundation** [comment]
what did you do to get them to work?

**Comment by arutar** [comment]
Nothing. It looks like my video card already had an updated firmware

**Comment by arutar** [comment]
[firmware.zip](https://github.com/arutar/FrankenDriver/files/10290198/firmware.zip)

**Comment by arutar** [comment]
@dynamo-foundation Have you flashed your video card?

**Comment by dynamo-foundation** [comment]
Ya I tried with an Asus 3060m and it bricked it.  I saved the old VBIOS but after flashing it back it still doesn't work.  I have tons of these cards so it's not a huge deal, but would be nice if someone has a fix for this.  None of the techpowerup VBIOS seem to be relevant to my specific card which is made by pcwinmax.

For anyone using the 341A - make sure to get a proper 1.8v version - there are many on Amazon that say 1.8v but are really 3.3v and it requires a mod.  Check with a multimeter to be 100% sure.
